api/src/test/java/org/openmrs/ConditionTest.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import org.junit.Test;
+
+/**
+ * Behavior-driven tests of the Concept class.
+ */
+public class ConditionTest {
+
+	@Test
+	public void constructorWithoutParameters_shouldInstantiateConditionClass(){
+		Condition condition = new Condition();
+		assert condition != null;
```

Comment:
Let is remove this test, for now, because it does not yet add much value.

Reply:
Okay